inspirehep/modules/migrator/tasks.py

```diff
@@ -277,7 +277,7 @@ def _get_records_to_update_generator(citations_lookup):
 
 def create_record(record):
     """Create record from marc21 model."""
-    json = overdo_marc_dict(record)
+    json = marcxml2json(record)
```

Comment:
Where is this defined?

Reply:
It will be defined in: inspirehep/inspire-dojson#157

Reply:
Actually there it will be called marc2record. Ouch!